### PR TITLE
Feature: added Result.run_id

### PR DESCRIPTION
**Pull request by marqueewinq**
Currently the insight package populates the Results without any grouping except created_at date. I'm adding `Result.run_id` to fix that.

### CODE REVIEW

**Comment by marqueewinq** [comment]
> [](https://sonarcloud.io/component_measures?id=synthesized-io_synthesized-insight&pullRequest=141&metric=new_coverage&view=list) [28.6% Coverage](https://sonarcloud.io/component_measures?id=synthesized-io_synthesized-insight&pullRequest=141&metric=new_coverage&view=list) [](https://sonarcloud.io/component_measures?id=synthesized-io_synthesized-insight&pullRequest=141&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=synthesized-io_synthesized-insight&pullRequest=141&metric=new_duplicated_lines_density&view=list)
> 

Perhaps we need to disable coverage for migrations

File: README.md [hunk 1]
```diff
@@ -43,6 +43,14 @@ or multiple dataframes.
...
 ```
 
+### Migrations
+
+`insight` populates the results to the Postgres database configured by environment variables. To run migrations against it, simply:
+
+```bash
+insight-migrations
+```
+
```
Comment (simonhkswan) [inline README.md:53]: 👍🏼

File: src/insight/alembic/main.py [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 
 here = os.path.dirname(os.path.abspath(__file__))
 
+args = sys.argv[1:]
+args = args if len(args) > 0 else ["upgrade", "head"] # default
 alembic_args = [
     '-c', os.path.join(here, 'alembic.ini'),
-    'upgrade', 'head'
-]
+] + args
```
Comment (simonhkswan) [inline src/insight/alembic/main.py:13]: Ah, nice way to update this.

File: src/insight/metrics/base.py [hunk 3]
```diff
@@ -68,7 +68,8 @@ def _add_to_database(self,
                          dataset_name: str,
                          dataset_rows: int = None,
                          dataset_cols: int = None,
-                         category: str = None):
+                         category: str = None,
+                         run_id: str = None):
```
Comment (simonhkswan) [inline src/insight/metrics/base.py:72]: Given that `version` is obtained from the an envrionment variable, does it make sense for run_id to also be obtained in the same way? (See lines 83:84 of this file).

Also I've just noticed that line 81: should probably be removed and replaced with run_id if we leave `run_id` as an argument.

**Review by simonhkswan — CHANGES_REQUESTED** [review]
I've left one question about the _add_to_database function. But the rest all looks great! Once we've addressed that one Q we can definitely merge.

**Review by simonhkswan — APPROVED** [review]
Nice work! Thanks for making that change.

**Pull request merged**